index.html

```diff
@@ -195,6 +195,32 @@ <h1>Skills</h1>
                 </div>
             </article>
         </section>
+        <section class="contact-form">
+            <article>
+                <div class="new-contact">
+                    <h1>Contact me</h1>
+                    <p id="new-info">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+                </div>
+                <div class="contact-fo">
+                    <form action="https://formspree.io/f/{form_id}" method="post">
+                        <ul>
```

Comment:
Kindly make sure that your form is working with Formspree :pray: :pray:. As the [last code reviewer suggested ](https://github.com/girma336/mobile-portfolio/pull/2#discussion_r987374809) first you need to create a new account in Formspree and configure it. You will get a URL like this "https://formspree.io/f/xhyyajk". This URL is what you need to put in your HTML code. As you can see in this URL the code "xhyyajk" is what you need to replace in the {form_id} :nerd_face: :nerd_face: This is just an example of a URL. You need to generate your own in Formspree.
![Screenshot from 2022-10-04 22-20-45](https://user-images.githubusercontent.com/98363075/193975665-d898014b-b13e-4dcc-bcb4-39a7dd441341.png)